openreview/venue/webfield/reviewersWebfield.js

```diff
@@ -1,10 +1,11 @@
 // Webfield component
+const committee_name = entity.id.split('/').slice(-1)[0].replaceAll('_', ' ')
 return {
   component: 'ReviewerConsole',
   version: 1,
   properties: {
     header: {
-      title: 'Reviewer Console',
+      title: `${committee_name} Console`,
       instructions: '<div><p>some instructions</p></div>'
```

Comment:
Can we add some default instructions here:

``<p class="dark">This page provides information and status updates for ${domain.content.subtitle?.value}. It will be regularly updated as the conference progresses, so please check back frequently.</p>``

Reply:
Done.